src/App.vue

```diff
@@ -1,10 +1,38 @@
 <template>
   <div id="app">
-    <div id="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
-    </div>
-    <router-view/>
+    <b-navbar toggleable="lg" type="dark" variant="info">
+      <b-container>
+        <b-navbar-brand href="#">Site Nav</b-navbar-brand>
+
+        <b-navbar-toggle target="nav-collapse"></b-navbar-toggle>
+
+        <b-collapse id="nav-collapse" is-nav>
+          <b-navbar-nav>
+            <b-nav-item href="#">Home</b-nav-item>
```

Comment:
This looks great! I ran into an issue with Vue Router using href properties. We'll need them all changed to `to` instead so that they continue to navigate through the page properly. `href` will be used for external links still.

Reply:
ok, I'll make the change.